From 6223549e4d5e1d442a9cfb8e8c7334bcc62c1662 Mon Sep 17 00:00:00 2001
From: Maarten de Vries <maarten@de-vri.es>
Date: Thu, 17 Jan 2019 16:00:14 +0100
Subject: [PATCH] Remove `signals` from find_package(Boost COMPONENTS ...).

tf2 is using signals2, which is not the same library.
Additionally, signals2 has always been header only, and header only
libraries must not be listed in find_package.

Boost 1.69 removed the old signals library entirely, so the otherwise
useless `COMPONENTS signals` actually breaks the build.
---
 tf2/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tf2/CMakeLists.txt b/tf2/CMakeLists.txt
index 9ffb17a2..7a6da342 100644
--- a/tf2/CMakeLists.txt
+++ b/tf2/CMakeLists.txt
@@ -3,7 +3,7 @@ project(tf2)
 
 find_package(console_bridge REQUIRED)
 find_package(catkin REQUIRED COMPONENTS geometry_msgs rostime tf2_msgs)
-find_package(Boost REQUIRED COMPONENTS signals system thread)
+find_package(Boost REQUIRED COMPONENTS system thread)
 
 catkin_package(
    INCLUDE_DIRS include
